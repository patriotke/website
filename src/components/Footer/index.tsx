/**
 *
 * Footer
 *
 */

function Footer() {
  return (
    <div className="bg-gray-100 pb-[env(safe-area-inset-bottom)] dark:bg-neutral-900 print:bg-transparent">
      <hr className="dark:border-neutral-800" />
      <div className="mx-auto flex max-w-[90rem] justify-center py-12 text-gray-600 dark:text-gray-400 md:justify-start pl-[max(env(safe-area-inset-left),1.5rem)] pr-[max(env(safe-area-inset-right),1.5rem)]">
        <div className="flex w-full flex-col items-center sm:items-start">
          <div>{process.env.NEXT_PUBLIC_TITLE}</div>
          <p className="mt-6 text-xs">{new Date().getFullYear()}</p>
        </div>
      </div>
    </div>
  );
}

export default Footer;
