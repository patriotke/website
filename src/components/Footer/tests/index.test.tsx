/* eslint-disable import/no-extraneous-dependencies */
import '@testing-library/jest-dom';
// eslint-disable-next-line @typescript-eslint/no-unused-vars
import { render, screen } from '@testing-library/react';
import Footer from '../index';

describe('Footer', () => {
  it('renders Footer', () => {
    render(<Footer />);

    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
