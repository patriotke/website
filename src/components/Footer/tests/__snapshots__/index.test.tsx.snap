// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders Footer 1`] = `
<div>
  <div
    class="bg-gray-100 pb-[env(safe-area-inset-bottom)] dark:bg-neutral-900 print:bg-transparent"
  >
    <hr
      class="dark:border-neutral-800"
    />
    <div
      class="mx-auto flex max-w-[90rem] justify-center py-12 text-gray-600 dark:text-gray-400 md:justify-start pl-[max(env(safe-area-inset-left),1.5rem)] pr-[max(env(safe-area-inset-right),1.5rem)]"
    >
      <div
        class="flex w-full flex-col items-center sm:items-start"
      >
        <div>
          App
        </div>
        <p
          class="mt-6 text-xs"
        >
          2024
        </p>
      </div>
    </div>
  </div>
</div>
`;
